grid-demo/src/app/angular/column-ordering/column-ordering.component.html

```diff
@@ -1,3 +1,31 @@
+<clr-accordion>
+  <clr-accordion-panel [clrAccordionPanelOpen]="true">
+    <clr-accordion-title>Demo Explanation</clr-accordion-title>
+    <clr-accordion-content>
+      <p>
+        This demo allows users to order columns. There are two ways to order columns. One option is to drag and drop the
+        columns with the mouse. Another option is to focus on the column with the keyboard, press space to grab the
+        column, use the keyboard arrows to move the column, and finally, press space again to drop the column.
+      </p>
+
+      <p>
+        This behavior can be added to an angular datagrid by adding the new
+        <code>customClrColumnOrderingGrid</code> directive and then angular cdk directive <code>cdkDropList</code> to
```

Comment:
I know the directive is prefixed with `custom` but it might be helpful to call out that this doesn't come from Clarity (yet, or possibly at all).